plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobPreview.qml

```diff
@@ -47,11 +54,32 @@ Item
 
     UM.RecolorImage
     {
-        id: statusImage
+        id: overlayIcon
         anchors.centerIn: printJobPreview
         color: UM.Theme.getColor("monitor_image_overlay")
         height: 0.5 * printJobPreview.height
-        source: printJob && printJob.state == "error" ? "../svg/aborted-icon.svg" : ""
+        source:
+        {
+            switch(printJob.state)
+            {
+                case "error":
+                    return "../svg/aborted-icon.svg"
+                case "wait_cleanup":
+                    if (printJob.state == "wait_cleanup" && printJob.timeTotal > printJob.timeElapsed)
+                    {
+                        return "../svg/aborted-icon.svg"
+                    }
+                    break;
+                case "pausing":
+                    return "../svg/paused-icon.svg"
+                case "paused":
+                    return "../svg/paused-icon.svg"
+                case "resuming":
+                    return "../svg/paused-icon.svg"
+                default:
+                    return ""
+            }
+        }
```

Comment:
This function can return `undefined` when it should probably return something else like "".

Reply:
Ummmm can it? The switch is the first block and it has a default case at the end to ensure it always returns a string.